lib/ovh.es6.js

```diff
@@ -1,6 +1,6 @@
 /**
- * Copyright (c) 2014 OVH SAS
- * Copyright (c) 2012 - 2013 Vincent Giersch
+ * Copyright (c) 2016 OVH SAS
+ * Copyright (c) Benjamin Coenen
```

Comment:
Not sure we need this line. Technically, the copyright belongs to OVH. Maybe both lines could be merged ?